cdc/cdc.cc

```diff
@@ -64,6 +64,24 @@ using namespace std::chrono_literals;
 
 static logging::logger cdc_log("cdc");
 
+class cdc::cdc_service::impl {
```

Comment:
Why not place this below inside namespace cdc and drop cdc::? There's already code there. It would be more consistent to do it in one way.